### PR TITLE
add more detail to the README

### PR DESCRIPTION
**Pull request by JamesBie**
reorder text
clarify instruction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,21 +38,23 @@ Follow these steps to run the bot and do development on your local machine.
 
 Pre-requisites: A Discord account.
 
-1. Create your own Discord server for testing
+Create your own Discord server for testing
```
Comment (modernNeo) [inline README.md]: why are you removing the number here?

**Comment by modernNeo** [comment]
@JamesBie looks good, just waiiting on someone else to determine why the checks arent clearing and then I will approve this.

**Comment by wnfldchen** [comment]
I believe this happens when the PR is from a forked repo, as Jenkins doesn't read branches from forked repos - I think.

**Comment by modernNeo** [comment]
@JamesBie please merge PR, I want to see if the requestor can merge it

**Comment by wnfldchen** [comment]
I had the same idea, he actually can't there's no green "Merge pull request" button on his screen

**Pull request merged**